modules/vulnerabilities/unix/http/apache_couchdb/secgen_metadata.xml

```diff
@@ -0,0 +1,71 @@
+<vulnerability xmlns="http://www.github/cliffe/SecGen/vulnerability"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.github/cliffe/SecGen/vulnerability">
+
+  <name>Apache CouchDB Unauthenticated RCE</name>
+  <author>Sofia Markusfeld</author>
+  <author>James Davis</author>
+  <module_license>Apache v2</module_license>
+  <description>
+    Apache CouchDB versions 3.2.1 and below default installation settings are vulnerable to an
+    Earling RCE exploit.
+  </description>
+
+  <type>remote</type>
+  <privilege>user_rwx</privilege>
+  <access>remote</access>
+  <platform>linux</platform>
+  <difficulty>low</difficulty>
+
+  <read_fact>port</read_fact>
+  <read_fact>known_username</read_fact>
```

Comment:
These listed parameters have to match the ones for default inputs below. If you read_fact `known_username`, then it should/can also have a default input.